targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Source/ExampleProject/PlayFabApiTests.h

```diff
@@ -421,7 +431,9 @@ class PlayFabApiTestSuite : public FAutomationTestBase
 
     bool InvalidLogin() const
     {
-        ADD_LATENT_AUTOMATION_COMMAND(PlayFabApiTest_0LoginWithEmail(testTitleData.userEmail, INVALID_PASSWORD));
+        if (!testTitleDataStorage.loaded)
+            return false;
```

Comment:
If RunTest is called for each test function, then this check might not be necessary? (RunTest already has it)

Reply:
You're right - totally redundant.  Will remove.